docs/source/contributing/translation.rst

```diff
@@ -0,0 +1,50 @@
+====================
+Internationalization
+====================
+
+If you want to help with translating the docs:
+
+#. Make sure you have a `Transifex <https://www.transifex.com/>`_ account. Transifex is a tool for colaborative translation and allows the Manim team to review translations.
+#. Open the `Manim Transifex project page <https://www.transifex.com/manim-community/manim-community-documentation>`_ .
+#. Click on ``Join Team``.
+#. Select the language you want to help with.
+
+Congratulations! You are now a Translator for the Manim project!
+
+
+You have access to the projects you joined just on the left of your profile picture
+on the top bar.
+
+.. image:: /_static/select_transifex_repo.png
+   :align: center
+   :width: 200
+
+Now go to the
+`languages list <https://www.transifex.com/manim-community/manim-community-documentation/languages/>`_
+and select a language you want to help with. You should see a blue button reading
+`Translate`; click it to get started! You will see a list of
+resources in the project. You can click on any untranslated resource, then
+click on `Translate`. You will then see something like the following:
+
+.. image:: /_static/transifex_explanation.png
+    :align: center
+
+The `A` panel is the list of all the strings in the resource. Click on any of
+them to translate it. It will appear in the `B` panel, and you will be able to
+work on your translation in the `C` panel. Once you're done, click `Save changes`.
+Your translation will have to be reviewed before being added to the docs.
+You can also get back to the resources list by clicking `d` or change the
+destination language in `e`. Your
+
+
+If you want to become a reviewer, please join `Manim's Discord server <https://www.manim.community/discord/>`_
+and you can ping the role @i18nService.
```

Comment:
I feel like a name like "Translation Admin" would be better

Reply:
Instead of i18nService ? You're right. How should i write the spaces ? @TranslationAdmin or @Translation Admin

Reply:
The latter